bin/askx-smapi.js

```diff
@@ -16,7 +16,7 @@ if (!process.argv.slice(2).length) {
             Messenger.getInstance().info(message);
         })
         .catch(err => {
-            Messenger.getInstance().fatal(err.message);
```

Comment:
personally I don't agree with this change. Warn typically means something may work but no 100% to the expectations. In our situation, the command failed, so should be Fatal or no prefix at all.

Reply:
No not Messenger.warn, but Messenger.error. 
So `error` method is used to tell users that you mistype something wrong and need fix. `fatal` is used to denote okay our side has something wrong, please report and let us know.
The color for warn, error and fatal are yellow, red, dark red. It's showing the severeness lol.